sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/VirtualMachine.java

```diff
@@ -2626,7 +2656,8 @@ interface Update extends Appliable<VirtualMachine>, Resource.UpdateWithTags<Upda
         UpdateStages.WithBillingProfile, UpdateStages.WithSystemAssignedManagedServiceIdentity,
         UpdateStages.WithUserAssignedManagedServiceIdentity, UpdateStages.WithLicenseType,
         UpdateStages.WithAdditionalCapacities, UpdateStages.WithOSDisk, UpdateStages.WithSecurityFeatures,
-        UpdateStages.WithDeleteOptions, UpdateStages.WithSecurityProfile, UpdateStages.WithUserData {
+        UpdateStages.WithDeleteOptions, UpdateStages.WithSecurityProfile, UpdateStages.WithUserData,
+        UpdateStages.WithCapacityReservation {
```

Comment:
Does `copyInnerToUpdateParameter` need to be updated for the update path to correctly set the capacity reservation?

Reply:
Sorry for not noticing the particularity of this service update process. Now fixed it in the [commit#94f34b6](https://github.com/Azure/azure-sdk-for-java/pull/42914/commits/94f34b6b7bd61890abc7b590ecd66af4639d78d8)

Reply:
Sorry for not noticing the particularity of this service logic. Now fixed it in the [commit#b27909e](https://github.com/Azure/azure-sdk-for-java/pull/42914/commits/b27909ec61193d4031efb3c408de078cd4b3914e).